nncf/experimental/onnx/graph/nncf_graph_builder.py

```diff
@@ -48,7 +51,8 @@ def _is_valid_onnx_metatype(node: NodeProto) -> bool:
         node_type = node.op_type
         metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node_type)
         if metatype == ONNXConstantMetatype:  # We don't need to quantize Constants
-            nncf_logger.debug('The metatype is ONNXConstantMetatype, which in not quantizable. Skipping this node.')
+            nncf_logger.debug('The metatype is ONNXConstantMetatype, which means that the node is Constant.'
+                              'All constant nodes are not added to NNCFGraph. The node is skipped.')
```

Comment:
Why do you need this debug message?

Reply:
For debugging purpose. I feel it could be helpful. What do you think?